### ISSUE TITLE
Unable to delete photo's in the photobooth_photos folder

### ISSUE THREAD

**Issue by hermantherock** [jrleeman/PiBooth#8]
When I select a photo and try to delete (move to trash) - I get a Permission Denied error message. 
I know very little about Linux and normally use Windows , I have been following the Magazine article and had to play with a few things and got the PhotoBooth to fire up! Great project, really gets me interested in all this stuff !!

**Comment by jrleeman** [comment]
Hi @hermantherock - Glad you are enjoying the project! The issue here I think is that the files were created by the root user, but you don't have permissions to delete them. You could remove them from the command line:

(removes all photos in the directory): `sudo rm *.jpg`

but that's a bit of a nuclear powered flyswatter. Better to give yourself permissions. On the command line go to the folder above the photos `cd /home/pi` then `sudo chmod u+w photobooth_photos` should do it. Let me know!

**Comment by hermantherock** [comment]
@jrleeman I was able to use the `sudo rm *.jpg` to clear out all the photos from the photobooth_photos folder successfully, thanks. 

I didn't have any luck using the command to change the folder permissions and delete them using the GUI, I kept getting the permission denied error.  

I am using the latest version of Raspbian Stretch this time around for running the Photobooth project, and I had to comment out `camera.resolution` (kept giving an error when I ran code) and I was unable to find the `/etc/inittab` file to edit, so I only added the commands to run program to the `/etc/profile` and it starts up the Photobooth code automatically when it boots with just that. This version of Raspbian I have even got the Twitter part working, Last time I used the Jessie version and I kept getting authentication errors. The only other things I had to change were the screen size stuff and font sizes - my screen is only 800x600 - which is also the resolution all the photos end up being now.

It boots to Photobooth program and turns off with the button and posts to Twitter. Now I know how to delete the Photos in case the SD card fills up!  Just in time for my daughters Halloween party this year, last year I didn't get the auto-start and twitter part going, but we had fun. 

Thanks

(sorry if this is too long, I've never used github before this either)

**Comment by jrleeman** [comment]
This looks like it might do it as well: https://raspberrypi.stackexchange.com/questions/14556/unable-to-move-file-to-the-wastebasket-permission-denied

I need to update my OS and see what's changing. Would you mind filing an issue on the resolution and `/etc/inittab` issues?

So glad you are enjoying the Photo Booth! What's the twitter handle of your box?

**Comment by hermantherock** [comment]
I added the issues.

I read the suggestions in link about the deleting issue, I don't have the  "Open current folder as root" option in the Tools menu, just three other options:   
Open Current Folder in Terminal
Find Files
Run a Command in Current Folder...
Not the end of the world, I can just delete them from the command if I run out of storage.

 I am using [https://twitter.com/ericbroxson](url) right now for this, I do plan on getting it set up on a different account in the future, now that the twitter part works. I don't recall exactly what all those problems were but I think it had something to do with oAuth and twython when I tried this on Raspbian Jessie version last year - one thing that stuck out was when I installed twython with pip this time on the Stretch version of Raspbian is that it installed a few packages with oauth stuff and it seemed to take longer to complete. It might even have something to do with the version of twython at that point in time and not the Raspbian version. Anyway that part now works fine following the instructions! 
This Saturday (28) will be the Halloween party, so currently not too many pictures on the twitter account.
